packages/components-js/tests/unit/fixtures/stats.json

```diff
@@ -11,8 +11,8 @@
   },
   {
     "chunkShortName": "button",
-    "size": 7229,
-    "gzipSize": 2894
+    "size": 7696,
```

Comment:
Sizing increased "heavily" for affected components, but why?

Reply:
I assume that this is caused by our `getHiddenTextJssStyle()` in combination with the new HTML node. Is there a possibility to extract this "global" styles to be rendered just once and not every time when it's used?